packages/next/README.md

```diff
@@ -2316,6 +2316,18 @@ To learn more about TypeScript checkout its [documentation](https://www.typescri
 > **Note**: Next.js does not enable TypeScript's `strict` mode by default.
 > When you feel comfortable with TypeScript, you may turn this option on in your `tsconfig.json`.
 
+> **Note**: By default, Next.js reports all TypeScript errors during dev and prevents application build if any are detected.
```

Comment:
This isn't correct, Next.js only reports TypeScript errors for pages that are being actively developed.

Reply:
@Timer good point! How would you prefer to phrase it? My slightly amended option:

> By default, Next.js reports TypeScript errors during development and also prevents application build if any are found.

Reply:
I'd like to specifically call out that only active pages are being reported.

> By default, Next.js will report TypeScript errors during development for pages you are actively working on. TypeScript errors for inactive pages do not block the development process.

Reply:
How about this?

> By default, Next.js will report TypeScript errors during development for pages you are actively working on. TypeScript errors for inactive pages do not block the development process. Trying to run `next build` for an app that has TypeScript errors will fail. If you don't want to leverage this behaviour and prefer to do type checks manually, set the following option in your `next.config.js`:

Reply:
The first option is good. The alternative does not mention the most important part 😄.

Reply:
Should be resolved, but feel free to suggest further improvements to the phasing 😉